cogs/commands/f1.py

```diff
@@ -0,0 +1,172 @@
+from datetime import timedelta
+
+from discord.ext import commands
+from discord.ext.commands import Bot, Context
+from requests import get as get_request
+from tabulate import tabulate
+
+LONG_HELP_TEXT = """
+
+Display F1 Statistics from current and past races
+
+Session Names: 
+    Practice 1, Practice 2, Practice 3, Qualifying, Race (Normal Format)
+    Practice 1, Qualifying, Sprint Shootout, Sprint, Race (Sprint Format)
+
+"""
+
+SHORT_HELP_TEXT = """Make the bot repeat after you."""
+
+
+base_url = "https://api.openf1.org/v1"
+
+
+class F1(commands.Cog):
+    def __init__(self, bot: Bot):
+        self.bot = bot
+
+    @commands.hybrid_command(help=LONG_HELP_TEXT, brief=SHORT_HELP_TEXT)
+    async def f1(
+        self,
+        ctx: Context,
+        country: str = None,
+        year: int = None,
+        session_name: str = None,
+    ):
+        if country is None or year is None:
+            raise Exception("Country or Year is Missing!")
+        session_key = self.get_session(country, year, session_name)
+        standings = await self.get_standings(session_key)
+        standings = await self.standings_format(standings, session_key)
+
+        standings = (
+            self.get_meeting(country, year)["meeting_official_name"]
+            + "\n"
+            + str(tabulate(standings))
+        )
+
+        await ctx.send(f"```{standings}```")
+
+    def get_meeting(self, country=None, year=None):
+        if country is None or year is None:
+            request = get_request(
```

Comment:
Could you replace these requests with non-blocking async requests? iirc there is a function to async fetch json in utils